.drone.yml

```diff
@@ -0,0 +1,18 @@
+build:
+  image: gregway/gbm_immune_validation:latest
```

Comment:
I think you should also use a version tag in addition to `latest` on Docker hub https://hub.docker.com/r/gregway/gbm_immune_validation/tags/. This way we know at any given state of your github repo, which version of the docker image to use. You will be responsible for not overwriting.

Currently, if you update the Docker and overwrite latest, someone may not be able to get back to the old computational environment.